Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -297,55 +297,64 @@ QVBoxLayout* SkylanderPortalWindow::CreateFinderLayout()
   auto* search_radio_layout = new QHBoxLayout();
 
   auto* radio_layout_left = new QVBoxLayout();
-  for (int i = 0; i < 10; i += 2)
-  {
-    QRadioButton* radio = new QRadioButton(this);
-    radio->setProperty("id", i);
-    connect(radio, &QRadioButton::toggled, this, &SkylanderPortalWindow::RefreshList);
-    m_element_filter[i] = radio;
-    radio_layout_left->addWidget(radio);
-  }
-  search_radio_layout->addLayout(radio_layout_left);
-
   auto* radio_layout_right = new QVBoxLayout();
-  for (int i = 1; i < 10; i += 2)
+  for (int i = 0; i < std::ssize(m_element_filter); i++)
```

Comment:
I think instead of using std::ssize here, you could instead add a constexpr in Skylander.h for NUM_SKYLANDER_GAMES, same for elements and types, and use that throughout the window methods just so it's clear the number that you are iterating over